auth/auth.go

```diff
@@ -205,11 +205,21 @@ func NewAuthenticator(ctx context.Context, c *Config) (*Authenticator, error) {
 }
 
 func newUnstartedAuthenticator(c *Config) (*Authenticator, error) {
-	client, err := newHTTPClient(c.IssuerCA, true)
+	// make sure we get a valid starting client
+	fallbackClient, err := newHTTPClient(c.IssuerCA, true)
 	if err != nil {
 		return nil, err
 	}
 
+	clientFunc := func() *http.Client {
```

Comment:
nit: can we undo the change in this PR that moved `newUnstartedAuthenticator` inside the for loop?

https://github.com/openshift/console/pull/1206/files#diff-3ba8fa87da15d764efcb8ea0c83f6e97L143